.github/ISSUE_TEMPLATE/proposal.yml

```diff
@@ -0,0 +1,40 @@
+name: Proposal
+description: Proposal for feature to include in Interop 2023
+labels: proposal
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to make a proposal for Interop 2023!
+
+      Please see the [Interop 2023 README](https://github.com/web-platform-tests/interop/blob/main/2023/README.md) for the timeline and what to expect from the process.
+- type: textarea
+  attributes:
+    label: Description
+    description: Brief description of the feature. Try to be clear about what parts of the feature and included and not.
+- type: textarea
+  attributes:
+    label: Specification
+    description: |
+      Specification for the feature. The feature should be defined by a standards-track specification from the following organizations:
+
+      - [IETF](https://www.ietf.org/) (Standard Track)
+      - [Khronos Group](https://www.khronos.org/)
+      - [TC39](https://tc39.es/) (Stage 2+)
+      - [W3C](https://www.w3.org/) (Recommendation Track)
+      - [WHATWG](https://whatwg.org/)
+
+      Other specifications will be considered on a case-by-case basis.
```

Comment:
Per meeting discussions, remove this since anyone who might have a proposal that requires the exception, and which has a reasonable chance of being accpeted, is probably close enough to the project that they know why the other rules don't apply in this specific case.

Reply:
Will remove!